packages/mobile/src/backup/BackupIntroduction.tsx

```diff
@@ -31,18 +33,29 @@ interface DispatchProps {
   exitBackupFlow: typeof exitBackupFlow
 }
 
-type Props = StateProps & DispatchProps
+type NavigationProps = StackScreenProps<StackParamList, Screens.BackupIntroduction>
+
+type Props = StateProps & DispatchProps & NavigationProps
 
 const mapStateToProps = (state: RootState): StateProps => {
   return {
     backupCompleted: state.account.backupCompleted,
   }
 }
 
-export const navOptionsForAccount: StackNavigationOptions = {
-  ...drawerHeader,
-  headerTitle: '',
-  headerRight: () => <DelayButton />,
+export const navOptionsForAccount = ({ route }: NavigationProps) => {
+  if (route.params?.fromAccountScreen) {
```

Comment:
would this be better as a more generic "showBackButton" or something?

Reply:
yeah good call